lua_protobuf/generator.py

```diff
@@ -348,36 +351,36 @@ def field_get(package, message, field_descriptor):
         # for scalar fields, we push nil if the value is not defined
         # this is the Lua way
         if type == FieldDescriptor.TYPE_STRING or type == FieldDescriptor.TYPE_BYTES:
-            lines.append('string s = m->%s();' % name)
-            lines.append('m->has_%s() ? lua_pushlstring(L, s.c_str(), s.size()) : lua_pushnil(L);' % name)
+            lines.append('string s = m->%s();' % name.lower())
+            lines.append('if (m->has_%s()) lua_pushlstring(L, s.c_str(), s.size()); else lua_pushnil(L);' % name.lower())
```

Comment:
Nit: Please don't make unrelated changed to code structure in the same commit as a substantive change, as this makes it more difficult to perform code review and to bisect failures later. Please revert back to the ternary operator or move this to its own commit.

Also, is `if (expr) statement; else statement;` valid syntax?! I don't think I've ever seen that. I don't like blocks without `{ ... }` because of things like the infamous "goto fail" bug. So if you are going to change this, please use common syntax.